src/utils/confighandler.cpp

```diff
@@ -446,6 +473,36 @@ void ConfigHandler::setContrastOpacity(const int transparency)
     m_settings.setValue(QStringLiteral("contrastOpacity"), transparency);
 }
 
+bool ConfigHandler::copyAndCloseAfterUploadEnabled()
```

Comment:
@borgmanJeremy Remove after merge since flameshot-org version removed close after capture due to instability.

Reply:
@borgmanJeremy It's up to you, but overall it works well and is one of the most requested features at Namecheap Customer Support team. Sometimes I have issue reports about it, but very rare. I have to notice, that CS Team are mostly Windows and Ubuntu users.